external_distributions/README.md

```diff
@@ -63,21 +63,23 @@ If you cannot obtain a recent enough version of CMake via your favourite package
 
 ## Library versions
 
-The table below contains the version of each library that you can install with this repository. If you wish to provide your own version of OpenBLAS or Boost, make sure it is the right version.
-
-| Library                          | Version                                                                                                                                   |
-|----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
-| `OpenBLAS` (**required**)        | [0.3.25](https://github.com/OpenMathLib/OpenBLAS/tree/v0.3.29)                                                                            |
-| `Boost` (**highly recommended**) | [1.83.0](https://github.com/boostorg/boost/tree/boost-1.83.0)                                                                             |
-| `CGAL` (**highly recommended**)  | [6.0.1](https://github.com/CGAL/cgal/tree/v6.0.1)                                                                                         |
-| `GKlib`                          | [commit `6e7951358fd896e2abed7887196b6871aac9f2f8`](https://github.com/KarypisLab/GKlib/tree/6e7951358fd896e2abed7887196b6871aac9f2f8)    |
-| `METIS`                          | [commit `a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2`](https://github.com/KarypisLab/METIS/tree/a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2)    |
-| `ParMETIS`                       | [commit `83bb3d4f5b2af826d0683329cad1accc8d829de2`](https://github.com/puneetmatharu/ParMETIS/tree/83bb3d4f5b2af826d0683329cad1accc8d829de2) |
-| `SuperLU`                        | [v6.0.1](https://github.com/xiaoyeli/superlu/tree/v6.0.1)                                                                                 |
-| `SuperLU_DIST`                   | [v9.1.0](https://github.com/xiaoyeli/superlu_dist/tree/v9.1.0)                                                                            |
-| `MUMPS`                          | [5.6.2](https://github.com/puneetmatharu/mumps/tree/v5.6.2.5)                                                                             |
-| `HYPRE`                          | [2.32.0](https://github.com/hypre-space/hypre/tree/v2.32.0)                                                                               |
-| `Trilinos`                       | [16.0.0](https://github.com/trilinos/Trilinos/tree/trilinos-release-16-0-0)                                                               |
+The table below lists the version of the third party libraries that work with(and/or are required by) `oomph-lib` and indicates which ones we can install for you via CMake. By default, we build "everything we can"; see below for how to customise the build process and select a serial or an MPI build. If you wish to provide your own version of these libraries, make sure it is the right version.
```

Comment:
> see below for how to customise

Hmm, I'm guessing you're not referring to the table here. Maybe "see further below" might be clearer?